### PR TITLE
Make minor improvements and fixes to Portfolio

### PR DESCRIPTION
**Pull request by zachlatta**

### CODE REVIEW

File: workshops/portfolio/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ And now we're done creating a GitHub account!
 
 Now that we have a GitHub account:
 
-1. Open [jsbin.com][jsbin]
+1. Open [jsbin.com][jsbin] in a new window
```
Comment (paked) [inline workshops/portfolio/README.md]: Have we added a note on "how" to do this?

File: workshops/portfolio/README.md [hunk 2]
```diff
@@ -248,7 +246,7 @@ side, we see that our original text is still there:
 
 > ![](img/scroll_down.gif)
 
-Ok. We need to make the picture smaller
+Ok. Let's make the picture smaller
```
Comment (paked) [inline workshops/portfolio/README.md]: Pretty sure this should be "lets"

http://public.wsu.edu/~brians/errors/lets.html

File: workshops/portfolio/README.md [hunk 3]
```diff
@@ -297,7 +295,7 @@ For example, if an oven was a tag:
 
 ## Changing the look and feel with CSS
 
-Ok. The image is way too big. We need to fix it.
+Ok. The image is way too big. Let's fix it.
```
Comment (paked) [inline workshops/portfolio/README.md]: again: http://public.wsu.edu/~brians/errors/lets.html

**Comment by paked** [comment]
(sorry for the dupe comments, I accidentally commented on the commit instead of diff)

File: workshops/portfolio/README.md (hunk 1 above)
Comment (zachlatta) [inline workshops/portfolio/README.md]: I don't think we need to, at least for the time being.

I was on a Google Hangout with a beginner yesterday and watched them while they went through the workshop. They didn't seem to have any trouble understanding this.

File: workshops/portfolio/README.md (hunk 2 above)
Comment (zachlatta) [inline workshops/portfolio/README.md]: I think that in this use case, it's actually standing for "let us".

Example that's expanded:

> Let us make the picture smaller

**Comment by zachlatta** [comment]
Just replied :-)

And totally no worries, thanks for taking the time to leave comments at all!

--
Zach Latta
Hack Club | zach@hackclub.io

File: workshops/portfolio/README.md (hunk 1 above)
Comment (jonleung) [inline workshops/portfolio/README.md]: I've seen many people run into the problem of:

1) Clicking on a link
2) Then the new page opens up in the same tab
3) Then they're not sure where the directions went and are unsure of how to go back to the directions

**Pull request merged**